### PR TITLE
Add test for history undo for install and erase

### PR DESCRIPTION
**Pull request by edynox**
Test should pass after this will be fixed https://bugzilla.redhat.com/show_bug.cgi?id=1463107

### CODE REVIEW

**Comment by edynox** [comment]
Other cases like update, reinstall or downgrade are handled in DNF unit tests.

File: dnf-docker-test/features/history_undo.feature [hunk 1]
```diff
@@ -0,0 +1,101 @@
+Feature: DNF/Behave test history undo
+
+  @setup
+  Scenario: Feature Setup
+      Given repository "test" with packages
```
Comment (ignatenkobrain) [inline dnf-docker-test/features/history_undo.feature]: `base`

File: dnf-docker-test/features/history_undo.feature [hunk 2]
```diff
@@ -0,0 +1,101 @@
+Feature: DNF/Behave test history undo
```
Comment (ignatenkobrain) [inline dnf-docker-test/features/history_undo.feature]: proper description is missing.. everyone knows that it is DNF test using behave and obviously everyone knows that this is a test

**Comment by j-mracek** [comment]
It looks fine and works. Thanks a lot.

**Pull request merged**